magma_access_gateway_installer/agw_installation_errors.py

```diff
@@ -29,6 +29,17 @@ def __init__(self):
         super().__init__("Invalid OS. Only Ubuntu 20.04 is supported.")
 
 
+class UnsupportedKernelVersionError(AGWInstallationError):
+    """Exception raised when unsupported kernel version is detected."""
+
+    def __init__(self):
+        self._message = (
+            "Invalid Kernel Version. Only 5.4.0 is supported. For downgrading kernel version, please refer to:\n"  # noqa E501, W505
+            "https://discourse.ubuntu.com/t/how-to-downgrade-the-kernel-on-ubuntu-20-04-to-the-5-4-lts-version/26459"  # noqa E501, W505
```

Comment:
Regarding the kernel version, Magma community says it should work with Kernel version `5.8` or lower. They've experienced issues with `5.8` as well. Should we keep `5.4` as the desired one?

Reply:
I took a look at OpenVswitch documentation yesterday, and found a list of supported kernels per version in the FAQ: https://docs.openvswitch.org/en/latest/faq/releases/. According to this, it should work with 5.8.

I know 20.04 shipped with 5.4, but some newer versions are now available. Are new installs coming with newer kernels by default?

Reply:
AWS `20.04` installs come with `5.13.0-1029-aws`. For instance, however the image from my Multipass VM, comes with `5.4.0-120-generic`. We might only have this issue for specific platforms.